src/reporter.js

```diff
@@ -101,20 +101,24 @@ class Reporter {
       browserName,
       browserVersion,
       platformName: this.getOsName(this.cypressDetails?.system?.osName),
-      saucectlVersion: 'v0.0.0',
     };
   }
 
-  async uploadAssets (sessionId, video, consoleLog, screenshots) {
+  async uploadAssets (sessionId, video, consoleLogContent, screenshots) {
     const assets = [];
 
     // Since reporting is made by spec, there is only one video to upload.
-    const videoPath = path.join(this.workDir, 'video.mp4');
-    fs.copyFileSync(video, videoPath);
-    assets.push(videoPath);
+    const videoContent = await readFile(video);
```

Comment:
I assume this can throw an exception. In which case we'd exit here and neither logs nor screenshots would be uploaded then. If my assumption is correct, could we perhaps harden this block a bit further, so that we gather the files we are able to?